packages/worker/src/workflow/bundler.ts

```diff
@@ -117,6 +117,7 @@ export class WorkflowCodeBundler {
           },
         ],
       },
+      context: path.resolve(__dirname, '../..'),
```

Comment:
This should usually be the directory where the app is located, not our SDK.
Can we `require.resolve` the `ts-loader` instead?

Reply:
@lorensr WDYT?

Reply:
@bergundy that's [what I had originally](https://github.com/temporalio/sdk-typescript/pull/492/commits/22ebb5c03a281c62b636cc34e02693a1906512d5) 😆

Reply:
Ah. Yeah, I guess that's better than having user provide their root dir to put for context. Sorry James 😄